.github/workflows/pr-stale.yml

```
@@ -0,0 +1,19 @@
+name: 'Mark inactive PRs as stale'
+on:
+  schedule:
+    # Update at 10:00 AM UTC+9
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity.'
+          # Mark PRs as stale if they are open and inactive for 60 days.
```

30 days already sounds too old to me - not sure about the others

```suggestion
          stale-pr-message: 'This PR is stale because it has been open 30 days with no activity.'
          # Mark PRs as stale if they are open and inactive for 30 days.
```